tikv-client/src/main/java/com/pingcap/tikv/util/Oracle.java

```diff
@@ -0,0 +1,15 @@
+package com.pingcap.tikv.util;
+
+final public class Oracle {
+  private static final long physicalShiftBits = 18;
+
+  public static boolean IsExpired(long lockTS, long ttl) {
```

Comment:
Why you have method start with uppercase?

Reply:
I am guessing `IsExpired` is the name of the method in tidb. uppercase is how golang let outside package access the method.